enterprise/backend/test/metabase_enterprise/serialization/test_util.clj

```diff
@@ -126,14 +126,14 @@
           (when (.exists (io/file dump-dir))
             (.delete (io/file dump-dir))))))))
 
-(defmacro with-random-dump-dir {:style/indent 2} [[dump-dir-binding prefix] & body]
+(defmacro with-random-dump-dir {:style/indent 1} [[dump-dir-binding prefix] & body]
```

Comment:
The correct indent spec for this was actually `1` since only the first arg is special.